code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1182,11 +1290,19 @@
 								if (ckey(param) == ckey(A.name))
 									M = A
 									break
-						if (M)
 #ifdef TWITCH_BOT_ALLOWED
 							if (IS_TWITCH_CONTROLLED(M))
 								return
 #endif
+						else
+							var/list/target_list = src.get_targets(1, "mob") // Chrunb Erbrbt and Scales To Lizard highfive!
+							if(length(target_list))
+								M = input(src, "Pick someone to high-five!", "EmotiConsole v1.1.3") as null|anything in target_list
+								if (M && !IN_RANGE(get_turf(src), get_turf(M), 1))
```

Comment:
you probably want interact dist (goes for all IN_RANGE checks)

Reply:
in_interact_range looks like it only checks the tiles right next to the player, so I made the ones that require physical contact use it, and left the ranged ones to IN_RANGE.